detox/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "detox",
   "description": "E2E tests and automation for mobile",
-  "version": "18.18.1",
```

Comment:
@noomorph be sure to unsmoke before merging in 😅

Reply:
@d4vidi, do I really need to? I don't think so. According to semantic version calculation, it is totally correct. prerelease → patch.